ui/src/containers/ClusterMonitoring.js

```diff
@@ -33,14 +33,13 @@ const PageContainer = styled.div`
   flex-direction: column;
   height: 100%;
   padding: ${padding.larger};
-
   .sc-loader {
     padding: 0 ${padding.smaller};
   }
 `;
 
 const TableContainer = styled.div`
-  height: 100%;
+  height: 80%;
```

Comment:
> For some reason, if we set the height of the table to 100% we can't see the last few rows.

If the component is from us, we should investigate why.
If it's a third-party library, we could report the bug to them.

Reply:
Yes! We should know the root problem.

We are planning to replace this table library actually.
Because it causes too much issue than the value it brings. 
We will do some research on the table.